README.md

```diff
@@ -2,36 +2,30 @@
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit&logoColor=white)](https://pre-commit.com/)
 [![Code style: black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-
 <!-- start elevator-pitch -->
-
 MOMAland is an open source Python library for developing and comparing multi-objective multi-agent reinforcement learning algorithms by providing a standard API to communicate between learning algorithms and environments, as well as a standard set of environments compliant with that API. Essentially, the environments follow the standard [PettingZoo APIs](https://github.com/Farama-Foundation/PettingZoo), but return vectorized rewards as numpy arrays instead of scalar values.
 
 The documentation website is at TODO, and we have a public discord server (which we also use to coordinate development work) that you can join here: https://discord.gg/bnJ6kubTg6.
-
 <!-- end elevator-pitch -->
 
 ## Environments
-
 MOMAland includes environments taken from the MOMARL literature, as well as multi-objective version of classical environments, such as SISL or Butterfly.
 The full list of environments is available at TODO.
 
 ## Installation
 <!-- start install -->
-
 To install MOMAland, use:
 ```bash
 pip install momaland
 ```
-
-This does not include dependencies for all families of environments (some can be problematic to install on certain systems). You can install these dependencies for one family like `pip install "momaland"` or use `pip install "momaland[all]"` to install all dependencies.
-
+This does not include dependencies for all components of MOMAland (not everything is required for the basic usage, and some can be problematic to install on certain systems).
+- `pip install "momaland[testing]"` to install dependencies for API testing.
+- `pip install "momaland[learning]"` to install dependencies for the supplied learning algorithms.
+- `pip install "momaland[all]"` for all dependencies for all components.
 <!-- end install -->
 
 ## API
-
 <!-- start snippet-usage -->
-
 As for PettingZoo, the MOMAland API models environments as simple Python `env` classes. Creating environment instances and interacting with them is very simple - here's an example using the "mosurround_v0" environment:
 
 ```python
```

Comment:
Why not tagging this and reusing on the API page? You already have divergence with the current way (duplication)

Reply:
Next commit will do this for AEC, but Parallel still needs to be done standalone since README only has one code snippet